client/src/components/LoginPage.js

```diff
@@ -96,7 +96,8 @@ class LoginPage extends Component{
   }
 }
   logIn(){
-    var currentUsername = this.state.username;
+    var lowerUsername = this.state.username.toLowerCase();
```

Comment:
i signed up with ABcd and logged in with abcd and it didnt seem to have a problem.